roles/init/tasks/postgres.yml

```diff
@@ -65,6 +65,13 @@
   become_user: root
   become: yes
 
+# There are cases where we must behave differently when pg_wal location
+# is inside pgdata. This fact is going to improve code clarity.
```

Comment:
```suggestion
# We must create postgres_wal_dir only when it's not in its default location under
# PGDATA, because initdb and pg_basebackup will be unhappy otherwise.
```